test-cases/performance/perf-row-level-repair-1TB.yaml

```diff
@@ -9,6 +9,8 @@ prepare_write_cmd: ["cassandra-stress write no-warmup cl=ALL n=125000000 -schema
                     "cassandra-stress write no-warmup cl=ALL n=125000000 -schema 'replication(factor=3)' -port jmx=6868 -mode cql3 native -rate threads=200 -col 'size=FIXED(1024) n=FIXED(1)' -pop seq=750000001..875000000",
                     "cassandra-stress write no-warmup cl=ALL n=125000000 -schema 'replication(factor=3)' -port jmx=6868 -mode cql3 native -rate threads=200 -col 'size=FIXED(1024) n=FIXED(1)' -pop seq=875000001..1000000000"]
 
+stress_cmd: "scylla-bench -workload=sequential -mode=write -max-rate=300 -replication-factor=3 -concurrency=10 -rows-per-request=30"
```

Comment:
why did you add it to this test case?

Reply:
This test relying on scylla-bench, so it needs to tell sct to install it